src/style.css

```diff
@@ -1 +1,104 @@
-/* add styles here */
+@font-face {
+  font-family: Avenir;
+  src: url('fonts/Avenir-Book.ttf');
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url('fonts/Avenir-Heavy.ttf');
+}
+
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+  padding: 0;
+  color: #3d4e61;
+  font-family: Avenir, sans-serif;
+}
+
+.big-form {
+  margin: 20px 15px;
+  position: relative;
+  border-radius: 4px;
+  border: 1px solid #e1e7ed;
+  background: #fff;
+  box-shadow: 0 1px 8px 0 rgba(61, 78, 97, 0.1);
+}
+
+.big-form:hover {
+  box-shadow: 0 4px 4px 0 rgba(0, 0, 0, 0.25);
+}
+
+.big-form:focus {
+  border-radius: 3px;
+  border: 1px solid #e1e7ed;
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+}
```

Comment:
The ':focus' pseudo-class is typically used on interactive elements like inputs, buttons, and links. Applying it to a '.big-form' div might not trigger as expected unless the div is made focusable (e.g., by adding a 'tabindex' attribute). Consider whether this is the intended behavior.